apps/server/src/modules/board/service/internal/board-node-copy-specific.service.spec.ts

```diff
@@ -224,51 +224,121 @@ describe(BoardNodeCopyService.name, () => {
 	});
 
 	describe('copy file element', () => {
-		const setup = () => {
-			const { copyContext } = setupContext();
-			const fileElement = fileElementFactory.build();
-			const fileCopyStatus: CopyFileDto = {
-				name: 'bird.jpg',
-				id: new ObjectId().toHexString(),
-				sourceId: new ObjectId().toHexString(),
+		describe('when copying is successful', () => {
+			const setup = () => {
+				const { copyContext } = setupContext();
+				const fileElement = fileElementFactory.build();
+				const fileCopyStatus: CopyFileDto = {
```

Comment:
Wenn nur in dem Test ein CopyFileDto benötigt wird, dann trotzdem als Funktion raus ziehen. Wird es woanders noch benötigt bitte eine mini test factory anlegen. Sollte sich das Format irgendwann mal ändern, möchten wir nur eine Stelle habe wo wir es anpassen müssen für alle Tests.